web/components/optional-lover-form.tsx

```
@@ -52,38 +64,102 @@ export const OptionalLoveUserForm = (props: {
-            className={'w-full sm:w-96'}
-            value={lover['twitter'] ?? undefined}
-          />
+        <Col className={clsx(colClassName, 'pb-4')}>
+          <label className={clsx(labelClassName)}>Socials</label>
+
+          <div className="grid w-full grid-cols-[8rem_1fr_auto] gap-2">
+            {Object.entries(newLinks)
+              .filter(([_, value]) => value != null)
+              .map(([platform, value]) => (
+                <>
+                  <div className="col-span-3 mt-2 flex items-center gap-2 self-center sm:col-span-1">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)